Makefile

```diff
@@ -66,12 +66,18 @@ test-all: test-unit
 .PHONY: test
 test: test-unit
 
+.PHONY: mock_setup
+mock_setup:
+	@echo Installing mock dependence:
+	go get -u github.com/matryer/moq
```

Comment:
What is the best practice of Go dev dependencies? I am not a big fan of adding another make command to install one dependency.

Reply:
We can remove this from the make file and have it just in the readme as an explanation.

Reply:
I am not sure how dev dependencies should be installed in Go. In Node.js it splits it logically in the `package.json`. Do you know how this should be done in Go?

Reply:
Done 👍